kasa/iot/modules/dimmer.py

```diff
@@ -0,0 +1,245 @@
+"""Implementation of the dimmer config module found in dimmers."""
+
+from __future__ import annotations
+
+import logging
+
+from ...exceptions import KasaException
+from ...feature import Feature
+from ..iotmodule import IotModule, merge
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Dimmer(IotModule):
+    """Implements the dimmer config module."""
+
+    def _initialize_features(self) -> None:
+        """Initialize features after the initial update."""
+        # Only add features if the device supports the module
+        if "get_dimmer_parameters" not in self.data:
+            return
+
+        self._add_feature(
+            Feature(
+                device=self._device,
+                container=self,
+                id="dimmer_threshold_min",
+                name="Minimum Dimming Level",
+                icon="mdi:lightbulb-on-20",
+                attribute_getter="threshold_min",
+                attribute_setter="set_threshold_min",
```

Comment:
```suggestion
                attribute_setter="set_threshold_min",
                range_getter=lambda x: (THRESHOLD_MIN, THRESHOLD_MAX)
```
Even when the inputs are checked in the setter, defining the allowed range for the feature allows our downstreams to adjust how they display this control :-)

Here's an example: https://github.com/python-kasa/python-kasa/blob/ba6d6560f4d7f8d23c81efed635501b7bbc736ee/kasa/iot/modules/light.py#L45-L46